util/package.go

```diff
@@ -107,6 +107,7 @@ type Owner struct {
 
 type Image struct {
 	Src   string `config:"src" json:"src" validate:"required"`
+	Path  string `config:"path" json:"path"`
```

Comment:
Does this change need a dependency update in integrations repository? Same question for package-storage.

Reply:
I don't think so as this only changes the output generated by the registry and nothing on the package side itself. Anything specific you were thinking of?

Reply:
I was thinking about the case in which the Kibana expects a missing property.

Reply:
This should be solved by https://github.com/elastic/kibana/pull/76434